.github/workflows/maven.yml

```diff
@@ -28,3 +30,13 @@ jobs:
     - name: Ensure code formatted with mvn spotless:apply
       working-directory: tiles
       run: mvn spotless:check
+    - name: Check for required version bump
+      run: |
+        CHANGED_FILES=$(git diff --name-only origin/${{ github.event.pull_request.base.ref }}...HEAD)
+        echo "$CHANGED_FILES"
+        TILES_SRC_CHANGED=$(echo "$CHANGED_FILES" | grep '^tiles/src/main' || true)
+        VERSION_CHANGED=$(echo "$CHANGED_FILES" | grep '^tiles/src/main/java/com/protomaps/basemap/Basemap.java$' || true)
+        if [[ -n "$TILES_SRC_CHANGED" && -z "$VERSION_CHANGED" ]]; then
```

Comment:
Can you move this into a check-version-bump.sh script? I think that is easier to maintain than CLI commands in github actions because you can test it locally...